src/lib/components/FloatingArrow/FloatingArrow.test.svelte.ts

```diff
@@ -1,12 +1,12 @@
 import { describe, expect } from 'vitest';
 import { render } from '@testing-library/svelte';
-import { it_in_effect } from '$lib/test-utils.svelte.js';
+import { testInEffect } from '$lib/test-utils.svelte.js';
```

Comment:
@Hugos68 If you and Malte are cooking up a solution, feel free to disregard my comment. But ideally I think we should aim to revert the tests cases and keep `it()` blocks in the test spec. The describe/it vernacular is important to test readability. It's the standard that third party contributors will understand.

Instead, I'd love to review if there's a built-in manner for wrapping each test case using `beforeEach` or similar.

Otherwise, I'd love to see you move towards the wrapping function call you described previously. I think it was something like this:

```ts
describe('xxx', () => {
    it('yyy', someWrapperFunction(() => {
        // (the code to test)
    }));
});
```

But either way, let's aim to keep the `it()` blocks visible here.

Reply:
I agree. The most reasonable way I've come up with is to either supply a function via context or to update the prototype of the `it` function. While the latter is possible I'd also like to go with the custom wrapper function. 
Or you go with a name like
```ts
itWithEffect('defaults to foo', /** ...*/);
```

Reply:
I totally agree, I went with the wrapper function so it now it's:
```js
it('...', withEffect(() => {
    // ...
}));
```